frontend/src/components/toast/toast-page-nav.js

```diff
@@ -15,7 +15,13 @@ class ToastPageNav extends React.Component {
   }
 
   handleClick(event) {
-    this.props.setStep(parseInt(event.target.getAttribute("name")));
+    if (this.props.enabled === undefined || this.props.enabled) {
+      this.props.setStep(parseInt(event.target.getAttribute("name")));
+    } else {
+      window.alert(
+        "Please fill out all required fields with an asterisk (*) beside it before continuing with the form."
```

Comment:
can you make the error message in red only show when this alert is shown (when they try to navigate to a different section of the form)? they haven't finished filling out the first page so there shouldnt be an error shown yet